erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -930,14 +934,16 @@ def update_item(source, target, source_parent):
 				["parent", "sales_order"],
 				["stock_uom", "stock_uom"],
 				["uom", "uom"],
-				["conversion_factor", "conversion_factor"],
-				["delivery_date", "schedule_date"]
```

Comment:
Seems it was correct.
"Expected delivery date" in SO can be set as "Required By Date" in PO